products-built-on-oxen/lokinet/guides/lokinet-on-a-soho-router.md

```diff
@@ -0,0 +1,325 @@
+# running lokinet client on a soho router
+
+the nicest way to run lokinet is on a soho router, it will make the defaults on the local network it serves use lokinet dns and even tunnel all traffic over lokinet in a way that it cannot leak.
+
+## requirements
+
+hardware with:
+
+* 1 wired interface 
+* 1 wireless interface
+* ability to run debian bookworm or later
+
+### network topology
+
+the network toplogy of this box will be the following:
```

Comment:
"For the sake of this document, we will assume the network toplogy of this box will be the following.  You may need to accommodate for different names/network addresses on your network:"

Reply:
Never mind, you mention this afterwards.